leanplum_api.gemspec

```diff
@@ -5,24 +5,26 @@ require 'leanplum_api/version'
 Gem::Specification.new do |gem|
   gem.name        = 'leanplum_api'
   gem.version     = LeanplumApi::VERSION
-  gem.authors     = ['Lumos Labs, Inc.']
-  gem.email       = ['analytics-dev@lumoslabs.com']
+  gem.authors     = ['Lumos Labs, Inc.', 'Countable Corp']
+  gem.email       = ['analytics-dev@lumoslabs.com', 'eng@countable.us']
   gem.homepage    = 'http://www.github.com/lumoslabs/leanplum_api'
   gem.summary     = 'Gem for the Leanplum API'
   gem.description = 'Ruby-esque access to Leanplum API'
   gem.licenses    = ['MIT']
   gem.files       = Dir["lib/**/*"] + ['Gemfile', 'LICENSE.txt', 'README.md']
   gem.test_files  = Dir["spec/**/*"]
 
-  gem.required_ruby_version = '>= 1.9.3'
+  gem.required_ruby_version = '>= 2.3'
 
   gem.add_dependency 'activesupport', '> 3.0', '< 5'
   gem.add_dependency 'awesome_print', '~> 1'
-  gem.add_dependency 'faraday', '~> 0.9', '>= 0.9.1'
-  gem.add_dependency 'faraday_middleware', '~> 0.10'
+  gem.add_dependency 'faraday', '~> 0.12', '>= 0.12.2'
```

Comment:
i can't accept this change; our app currently has faraday 0.9.2 and so this will be uninstallable

Reply:
ok. I'll bump it back down.